tests/RateLimitErrorTest.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Stripe;
+
+class RateLimitErrorTest extends TestCase
+{
+    private function rateLimitErrorResponse()
+    {
+        return array(
+            'error' => array(),
+        );
+    }
+
+    /**
+     * @expectedException Error\RateLimit
+     */
+    public function testRateLimit()
```

Comment:
I don't think the second test is actually necessary, since we pass through whatever HTTP code we get in the response (400 is pre-version-gate, 429 post).